### ISSUE TITLE
Unable to create Non-Workshop Instances

### ISSUE THREAD

**Issue by Hotmud9** [zetaPRIME/MultiBound2#9]
The option to make an instance without workshop mods doesn't appear.

**Comment by Doodlinka** [comment]
nothing about adding mods not from workshop either

**Comment by zetaPRIME** [comment]
yeah, I suppose I should probably add a flow for that; the steam workshop is just so all-encompassing that there barely seems to be anything outside of it at this point

**Comment by kassian0x0** [comment]
Any update on non steam mods?

**Comment by Lapidearum** [comment]
Yeah, as someone who has a bunch of mods that I've tweaked slightly for personal use, which I obviously am not going to be uploading to the Workshop, being able to control whether they're included in instances would be really nice.
